common/lua-system.hpp

```diff
@@ -13,29 +13,32 @@
 #include "tec-types.hpp"
 
 namespace tec {
-class LuaSystem;
-typedef Command<LuaSystem> LuaCommand;
-struct EntityCreated;
-struct EntityDestroyed;
+	class LuaSystem;
```

Comment:
Got some extra indent inside the namespace in this file

Reply:
Done